docs/coordinates/timezones.rst

```diff
@@ -0,0 +1,84 @@
+.. include:: references.txt
+
+.. _working_with_timezones:
+
+Working with Timezones
+**********************
+
+It is sometimes needed to find the timezone of an observer from their
+coordinates on Earth. Since timezones are defined legally for a site, they
+often don't simply follow the longitude of that site, as such, in order
+to figure out the timezone we need to consult a database.
+
+For this purpose we are going to use the IANA time zone database, which is
+part of Python's standard library with the `zoneinfo` module.
+
+.. _prerequisites:
+
+Prerequisites
+=============
+This functionality depends on the the `zoneinfo` module. This module is part
+of Python >= 3.9 standard library. In case you are using a lower Python version,
+you will need to install the `zoneinfo` module using the backports namespace
+package:
+
+.. code-block:: bash
+
+  pip install backports.zoneinfo
```

Comment:
We declare dependencies in `setup.cfg`.

Reply:
Thank you for the feedback.

I added the `>>>` prompt (although it impairs readability, are we supposed to use those everywhere?) and removed the explicit pip install of `backports`.

I don't know about the TOC. Maybe someone else should decide where this goes.

I also went through the style guide, by my poor grammar knowledge doesn't allow me to see where the style errors lie on the regard